pkg/plugins/generic/generic_plugin.go

```diff
@@ -163,8 +163,8 @@ func needDrainNode(desired sriovnetworkv1.Interfaces, current sriovnetworkv1.Int
 					needDrain = true
 					return
 				}
-				if iface.Mtu != 0 && iface.Mtu != ifaceStatus.Mtu {
-					glog.V(2).Infof("generic-plugin needDrainNode(): need drain, expect MTU %v, current MTU %v", iface.Mtu, ifaceStatus.Mtu)
+				if utils.NeedUpdate(&iface, &ifaceStatus) {
```

Comment:
this method checks also num VFs so you can remove the condition at L161

Reply:
done